tests/rest/client/test_relations.py

```diff
@@ -811,3 +860,65 @@ def _create_user(self, localpart: str) -> Tuple[str, str]:
         access_token = self.login(localpart, "abc123")
 
         return user_id, access_token
+
+    def test_background_update(self):
```

Comment:
Putting this in this file seems dirty, but it seems best to keep relations tests together? Maybe it would be better to start a storage test file and we can add to it in the future when fleshing out some unit tests?

Reply:
I think it's best here.